projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -43,6 +43,18 @@
 
 	const suggestRegenerate = suggestRegenerateDS.store;
 
+	$: if ( $modulesState.critical_css.active && $modulesState.critical_css.available ) {
+		resume();
+	} else {
+		alreadyResumed = false;
+	}
```

Comment:
This seems a bit awkward - ideally we should classify (wrap in a function?) the logic here - I haven't thought about this long and hard at the moment - but my gut reaction is "would this look awkward in React?". The goal is not only to move code around in the rafactor, but also to clean and package it up so that it's easier to port.

The awkward part is that in one, it's setting alreadyResumed to false, but in the other it's calling a function. This probably needs to invoke `resume` function with true/false and do the already resumed comparison there.

Reply:
Continued in the other review.